plugins/search/common/src/cc/search/common/ipc/IPCProcessor.java

```diff
@@ -22,8 +22,7 @@ public class IPCProcessor implements AutoCloseable {
   /**
    * Logger.
    */
-  private static final Logger _log  = Logger.getLogger(
-    IPCProcessor.class.getName());
+  private static final Logger _log  = Logger.getLogger("GLOBAL_LOGGER");
```

Comment:
Instead of ` Logger.getLogger("GLOBAL_LOGGER")`, the conventional way to use the global logger is `Logger.getGlobal()`. Please refactor all occurrences.

Reply:
That seems nicer indeed. Replaced them.